miniapps/solvers/block_fespace_operator.hpp

```diff
@@ -0,0 +1,60 @@
+#ifndef MFEM_BLOCK_FESPACE_OPERATOR
+#define MFEM_BLOCK_FESPACE_OPERATOR
+
+#include "mfem.hpp"
+
+namespace mfem
+{
+
+/// @brief Operator for block systems arising from different arbitrarily many finite element spaces.
```

Comment:
This will be good code to have to point users towards to explain how to handle BCs for block systems